### PR TITLE
New page for public workspaces

### PR DESCRIPTION
**Pull request by sowmyaraj92**
* To group public workspaces in a new page in the learning center

### CODE REVIEW

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    name: "Additional Resources"
+  - type: subtitle
+    name: "Related Blog Posts"
+  - type: link
+    name: "Creating workspaces"
+    url: "/docs/collaborating-in-postman/using-workspaces/creating-workspaces/"
+
+---
+
+[Public workspaces](https://blog.postman.com/public-workspaces-why-we-created-them-what-you-can-do/) enable you to browse and collaborate on entities with anyone across the world. Before you create a public workspace, navigate to team profile settings and enable your public team profile. This will ensure your team's profile will show up on the Public API Network.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:17]: I'd remove the "browse and" in the first sentence to make it "enable you to collaborate on entities" as I think it makes it a stronger statement

Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:17]: "navigate to team profile settings" -> "navigate to your [team profile settings](https://go.postman.co/settings/team/general)"

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+<img alt="Enable team profile" src="https://assets.postman.com/postman-docs/enable-team-profile.jpg" height="500px"/>
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:19]: Not a priority but I would crop this further on all sides to make the text clearer to read.

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+* [Leaving a public workspace](#leaving-a-public-workspace)
+* [Adding elements to a public workspace](#adding-elements-to-a-public-workspace)
+* [Removing elements from a public workspace](#removing-elements-from-a-public-workspace)
+* [Deleting a public workspace](#deleting-a-public-workspace)
+* [Managing public workspace members](#managing-public-workspace-members)
+* [Next steps](#next-steps)
+
+## Creating a public workspace
+
+There are four types of workspaces - personal, team, private and public workspaces. When you open a workspace, Postman will provide an overview of its contents, activity, and members in a new tab.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:36]: Add comma after "private"

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+Open the workspace menu, then click **+ New Workspace**.
+
+<img alt="New workspace" src="https://assets.postman.com/postman-docs/create-new-workspace-v8.jpg" width="400px"/>
+
+Specify a workspace name and summary. For __Free__ and __Team__ users, to create a public workspace change the workspace visibility to __Public__. Add collaborators by entering their email addresses, then define their [workspace roles](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles).
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:42]: I might add a blank line after the first sentence to separate it out (may help non-free-or-team users to scroll down the page a bit)

Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:42]: "For __Free__ and __Team__ users, to create a public workspace change the workspace visibility to __Public__. Add collaborators by entering their email addresses, then define their [workspace roles](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles)."

I'd rephrase a bit to something like the following if it sounds good

"If you are on a Postman __Team__ or __Free__ plan, you can create a public workspace by changing the workspace's visibility to __Public__. Add collaborators by entering their email addresses, then define their [workspace roles](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles)."

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+[![Public workspace](https://assets.postman.com/postman-docs/create-public-workspace-v8.jpg)](https://assets.postman.com/postman-docs/create-public-workspace-v8.jpg)
+
+You can also create a public workspace by selecting the team workspace and changing the team's visibility dropdown to __Public__ and clicking __Request to Change Visibility__.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:46]: I would adjust to something like

"You can also create a public workspace by selecting the team workspace, changing the team's visibility dropdown to __Public__, then clicking __Request to Change Visibility__."

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+
+[![Overview team workspace](https://assets.postman.com/postman-docs/overview-team-workspace.jpg)](https://assets.postman.com/postman-docs/overview-team-workspace.jpg)
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:48]: I'm not sure where to look for the above in this screenshot. Could you make it more obvious by cropping to where the action is and/or selecting?

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+
+For __Business__ and __Enterprise__ users, you need a community manager's approval to change the workspace visibility to __Public__.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:50]: I would rephrase this similar to the sentence for free/team above

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+
+<img alt="Visibility" src="https://assets.postman.com/postman-docs/visibility-options-v8.jpg" width="400px"/>
+
+Once you make a workspace public, the notification is sent to workspace members. Click the notification bell in the top right corner to view notifications.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:54]:
```suggestion
Once you make a workspace public, a notification is sent to workspace members. Click the notification bell in the top right corner to view notifications.
```

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+
+<img alt="In app notification" src="https://assets.postman.com/postman-docs/visibility-app-notification.jpg" width="500px"/>
+
+In addition to this, you will receive an email with the workspace information regarding who has made the change and which workspace was affected with the sharing link for the workspace.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:58]: Can we specify who "you" is here—all users or team admins, etc?

Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:58]: Maybe break off " with the sharing link for the workspace." to a separate sentence, like 

". The email will also provide a direct link to the workspace."

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+
+<img alt="Email notification" src="https://assets.postman.com/postman-docs/email-notification-public-workspace-v2.jpg" width="500px"/>
+
+Once you make a workspace public, all entities within that workspace become publicly accessible through the workspace. You need to be a [__workspace admin__](/docs/collaborating-in-postman/roles-and-permissions/#workspace-roles) to change the visibility of a workspace to public. Within a team, you cannot have two public workspaces with the same name.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:62]: I would put the workspace admin information higher up in this section. The two can't have the same name bit I would change into a note and keep down here.

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+
+[![Access Public Workspaces](https://assets.postman.com/postman-docs/accessing-public-workspaces.gif)](https://assets.postman.com/postman-docs/accessing-public-workspaces.gif)
+
+To view and edit your public workspace details (e.g. name, description), navigate to the workspace selector and hover over a workspace. Then, click **...** and select **View**.
+
+<img alt="Create public workspace dropdown" src="https://assets.postman.com/postman-docs/create-public-workspace-dropdown.jpg" height="350px"/>
+
+## Sharing a public workspace
+
+You can share a public workspace by sharing its URL directly. To access the public profile URL, navigate to **Your Team**, then click **View Team Settings**.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:78]: Can you explain where to get the URL for the workspace specifically after the first sentence?

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+
+If you are part of a team in Postman, you can find public workspaces to join. Open the workspaces drop-down from the control at the top of Postman in the left. Search for a public workspace and select the workspace you want to join.
+
+<img alt="Joining a public workspace overview" src="https://assets.postman.com/postman-docs/join-public-workspace-from-selector.jpg" height="350px"/>
+
+Select the public workspace, then click **Join Workspace** at the top right.
+
+<img alt="Join public workspace" src="https://assets.postman.com/postman-docs/join-public-workspace-from-overview-v8.jpg"/>
+
+Once you join the workspace, you can start creating collections, sending requests, visualizing your responses and even much more.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:92]:
```suggestion
Once you join the workspace, you can start creating collections, sending requests, visualizing your responses, and more.
```

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 13]
```diff
@@ -0,0 +1,166 @@
...
+
+## Leaving a public workspace
+
+To leave a public workspace, select the **...** on the right corner of the workspace overview and **Leave workspace**. Click **Leave** to finalize leaving the workspace.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:96]:
```suggestion
To leave a public workspace, select the **...** on the right corner of the workspace overview > **Leave workspace**. Click **Leave** to finalize leaving the workspace.
```

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 14]
```diff
@@ -0,0 +1,166 @@
...
+
+## Removing elements from a public workspace
+
+When you remove an element from a public workspace, it is no longer visible in that particular workspace. The element is still available in any other workspace where it has been added.
+
+From Postman, you can remove a collection from the sidebar. Click **...** to open the collection menu. Select **Remove from workspace** and confirm that you would like to remove the collection from the current workspace. The collection will no longer be visible in the workspace.
+
+<img alt="Removing a collection to public workspace" src="https://assets.postman.com/postman-docs/remove-collection-public-workspace.jpg" height="500px"/>
+
+Also, you can remove an environment from the sidebar. Click **...** next to the environment menu. You can also click **...** next to the Share button in the Environment tab. Select **Remove from workspace** and confirm that you would like to remove the environment from the current workspace. The environment will no longer be visible in the workspace.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:124]:
```suggestion
To remove an environment from the sidebar, click **...** next to the environment menu or click **...** next to the Share button in the Environment tab. Select **Remove from workspace** and confirm that you would like to remove the environment from the current workspace. The environment will no longer be visible in the workspace.
```

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 15]
```diff
@@ -0,0 +1,166 @@
...
+
+<img alt="Removing an environment to public workspace" src="https://assets.postman.com/postman-docs/remove-environment-public-workspace.jpg"/>
+
+> Note that deleting an element is different to removing it. When you delete an element it is no longer available in any workspace. When you remove an element from a specific workspace, it will still be available in any other workspaces it was in.
+
+## Deleting a public workspace
+
+Public workspaces must have their visibility reverted in order to be deleted. If you try to delete a public workspace by clicking **...** on the right corner of the workspace overview, the option to delete will be greyed out.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:132]: Instead of explaining the greyed out option, I'd give a quick review of how to change visibility, then delete, in this section

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 16]
```diff
@@ -0,0 +1,166 @@
...
+
+<img alt="Deleting a public workspace" src="https://assets.postman.com/postman-docs/delete-option-greyed-public-workspace.jpg"/>
+
+## Managing public workspace members
+
+If you're a team administrator you can manage the members of your public workspace, via the **Members** tab. You have the permission to add and remove any member from a public workspace. To add members to a public workspace, navigate to the Members and groups section, and click **Invite**.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:138]:
```suggestion
If you're a team administrator you can add or remove any member from your public workspace via the **Members** tab. To add members to a public workspace, click **Invite**.
```

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 17]
```diff
@@ -0,0 +1,166 @@
...
+
+<img alt="add workspace members public workspace (2)" src="https://assets.postman.com/postman-docs/manage-team-v8.jpg"/>
+
+Enter an email address, click **Add** to the team, select the roles you would like to assign to them, then select **Send Invitations**. You can also generate a link and invite people to your team by sharing the link with them.
+
+<img alt="Edit members for a public workspace" src="https://assets.postman.com/postman-docs/edit-member-public-workspace.jpg"/>
+
+> Team members will receive an email and in-app notification when added to a public workspace.
+
+To remove any team member, click the trash icon near the person you want to remove, and select **Remove User From Team**.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:156]:
```suggestion
To remove any team member, click the trash icon next to the individual you want to remove from the workspace, and select **Remove User From Team**.
```

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 18]
```diff
@@ -0,0 +1,166 @@
...
+
+If you're a team administrator you can manage the members of your public workspace, via the **Members** tab. You have the permission to add and remove any member from a public workspace. To add members to a public workspace, navigate to the Members and groups section, and click **Invite**.
+
+<img alt="add members for public workspace (1)" src="https://assets.postman.com/postman-docs/remove-team-member-workspace-overview.jpg" width="400px"/>
+
+Give a name, email or group name, click **Add** to add a team member to the workspace, then **Send Invites**.
+
+<img alt="add workspace members public workspace (3)" src="https://assets.postman.com/postman-docs/add-member-to-public-workspace.jpg"/>
+
+Another way to add workspace members is to click **Manage Team** and select **Invite Users**.
```
Comment (claireriley) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:146]: If not as important to have here, you could consider linking out to `/docs/administration/managing-your-team/managing-your-team/#inviting-to-a-team` and removing the steps here

**Review by claireriley — CHANGES_REQUESTED** [review]
Hey Sowmya! Looks good, suggested some edits. Make sure to pull in the new changes from develop so it doesn't overwrite my groups work.

File: src/pages/docs/administration/managing-your-team.md [hunk 19]
```diff
@@ -0,0 +1,88 @@
+---
```
[the remark this answers is not in the document]
Comment (sowmyaraj92) [reply src/pages/docs/administration/managing-your-team.md:1]: I remember pulling the latest version of develop. Sorry about that! :( 
Just did it again, I can see user group changes in the branch.

File: src/pages/docs/collaborating-in-postman/public-workspaces.md (hunk 6 above)
Comment (sowmyaraj92) [reply src/pages/docs/collaborating-in-postman/public-workspaces.md:48]: I will remove this screenshot and keep the one after Business and Enterprise users (zoomed-in bit).

**Comment by sowmyaraj92** [comment]
> Hey Sowmya! Looks good, suggested some edits. Make sure to pull in the new changes from develop so it doesn't overwrite my groups work.

> Hey Claire! Thank you for the detailed review. I have made the changes 👍

**Comment by sowmyaraj92** [comment]
`Managing your team` page is getting added every time I try to pull the latest version of develop. Ahhh! 😕 
Created a new PR #3012 to resolve this issue, so going ahead and closing this PR.

File: src/pages/docs/collaborating-in-postman/public-workspaces.md [hunk 20]
```diff
@@ -0,0 +1,171 @@
...
+
+To remove any team member, click the trash icon next to the individual you want to remove from the workspace, and select **Remove User From Team**.
+
+<img alt="Remove team members for a public workspace" src="https://assets.postman.com/postman-docs/remove-member-public-workspace.jpg"/>
+
+Click X next to the team member you want to remove from the public workspace, and click **Save**.
+
+<img alt="Remove members for a public workspace from overview" src="https://assets.postman.com/postman-docs/remove-team-member-workspace-overview.jpg" width="400px"/>
+
+## Next steps
```
Comment (avinashch95) [inline src/pages/docs/collaborating-in-postman/public-workspaces.md:169]: @sowmyaraj92 we could also add a section about adding categories to a public workspace from the profile page.
